ui/src/app/cohort-search/list-search/list-search.component.spec.tsx

```diff
@@ -16,7 +16,6 @@ beforeEach(() => {
   serverConfigStore.set({
     config: {
       ...defaultServerConfig,
-      enableDrugWildcardSearch: false,
     },
   });
```

Comment:
Remove the entire `serverConfigStore` reference

Reply:
done